client/components/jetpack-colophon/index.jsx

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import JetpackLogo from 'components/jetpack-logo';
+
+const JetpackColophon = ( { className } ) => {
+	return (
+		<div className={ classNames( 'jetpack-colophon', className ) }>
+			<span className="jetpack-colophon__power">Powered by </span><JetpackLogo size={ 24 } full />
```

Comment:
Are we not translating the `Powered by` string intentionally?

Reply:
➕  for translating.

We'd probably need to get the logo inside the `<span/>` for easier translation, so it can form part of the same phrase, i.e.

```jsx
<span>{ translate( 'Powered by {{jetpackLogo/}}', {
  components: {
    jetpackLogo: <JetpackLogo size={ 24 } full />,
  }
) }</span>
```

Is something like that possible?